docs/3-scaling/0-conduit-operator.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: "Conduit Operator"
+toc_max_heading_level: 6
+---
+
+The **Conduit Operator** is a [Kubernetes operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) designed to simplify the management and orchestration of Conduit instances. The operator extends the [Kubernetes API](https://kubernetes.io/docs/concepts/overview/kubernetes-api/) with a [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for Conduit. 
+
+This custom resource allows users to define and manage Conduit instances declaratively, enabling seamless integration with Kubernetes' native features such as scaling, monitoring, and logging.
+
+### Overview
+
+Conduit pipelines are represented as Conduit custom resources, where each pipeline will be provisioned 
+as a distinct Conduit instance with its own lifecycle. 
+
+The Conduit custom resource definition format is very similar to that of [pipeline configurations](/docs/using/pipelines/configuration-file):
+
+```yaml
+apiVersion: operator.conduit.io/v1
+kind: Conduit
+metadata:
+  name: conduit-generator
+spec:
+  running: true
+  name: generator.log
+  description: generator pipeline
+  connectors:
+    - name: source-connector
+      type: source
+      plugin: builtin:generator
+      settings:
+        - name: format.type
+          value: structured
+        - name: format.options
+          value: "id:int,name:string,company:string,trial:bool"
+        - name: recordCount
+          value: "3"
+    - name: destination-connector
+      type: destination
+      plugin: builtin:log
+```
+
+### Quickstart
+
+:::info
```

Comment:
Do we maybe have an example of getting started without cloning the repo and building the code? I think we should be able to use the Docker images?

Reply:
@hariso you're right. This one maybe? https://github.com/ConduitIO/conduit-operator/pkgs/container/conduit-operator

Reply:
@hariso At the moment, I'm linking to the README directly, so maybe we should update it there. Or do you think we should update it here? If we include a different set of instructions, it might make sense to add them here. My only concern is leaving stale information by not updating the same information in both places.

Reply:
Having it in the README is fine for me (we're following a slightly different principle with Condit itself, but I don't think it's a big thing). This way or the other, I believe the end result should be that we have instructions that don't require a user to clone the repo and build it.

Reply:
@hariso  @raulb I think using helm directly should work. Docker won't because the operator needs to live inside the Kubernetes cluster or have access to its API.

Since the operator requires a cluster, the easiest way to deploy it without building is to use the deployment with helm part, in the readme I just updated https://github.com/ConduitIO/conduit-operator/pull/47